src/map/battle.cpp

```diff
@@ -10276,6 +10276,11 @@ static const struct _battle_data {
 	{ "feature.dynamicnpc_rangey",          &battle_config.feature_dynamicnpc_rangey,       2,      0,      INT_MAX,        },
 	{ "feature.dynamicnpc_direction",       &battle_config.feature_dynamicnpc_direction,    0,      0,      1,              },
 
+	{ "feature.goldpc_active",              &battle_config.feature_goldpc_active,           1,      0,      1,              },
+	{ "feature.goldpc_time",                &battle_config.feature_goldpc_time,          3600,      0,   3600,              },
```

Comment:
Is it strange we cap this to 1 hour or is that because of a client limitation?

Reply:
Its a client limitation. Client always counts down from 3600. We can only fake the value by decreasing it. ;-)